config.yaml

```diff
@@ -20,16 +20,20 @@ logging:
 #    host: logstash1003.eqiad.wmnet
 #    port: 12201
 
-# Statsd metrics reporter
+# Metrics configuration
 metrics:
-  type: statsd
-  host: localhost
-  port: 8125
+  - type: prometheus
+    port: 9000
+  - type: statsd
+    host: localhost
+    port: 8125
 
 services:
   - name: test
     # a relative path or the name of an npm package, if different from name
-    module: ./some_server_module.js
+
```

Comment:
Remove the empy line and the comment below, no need for either of these to pollute the config.